poglink/models/rates.py

```diff
@@ -132,7 +134,7 @@ def from_statuses(cls, old, new):
     # highlights changed rates in embed
     def to_embed(self):
 
-        rates_dict = self.old.to_dict()
+        rates_dict = self.old.to_dict(include_extras=False)
```

Comment:
I was wrong about the suggested fix as mentioned in issue #37 .

That filter would prevent you from **bolding** any extra values, but it was actually this line which originally included those extra values in the first place.